src/gulp-plugins/gulp-mdtoXSLT.js

```diff
@@ -42,7 +42,7 @@ module.exports = function mdtoXSLT(options) {
     return through.obj(function(file, enc, callback) {
         var context = getContext(file);
         var t = this;
-        output(context);
+        // output(context);
```

Comment:
These were not entirely useful. Happy to back this out if you think we should leave them in.